packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -639,6 +639,7 @@ class DateTimePicker extends React.Component {
 
         <div className={cx('time-facade')}>
           <TimeInput
+            data-date-time-input
```

Comment:
The way this is being used in `TimeInput` does not make sense to me. Do you intend this to be a data attribute on a DOM element? I see that this is used as a 'flag' to indicate that `DateTimePicker` created `TimeInput`, is it meant to be just that?

Reply:
yes it was meant to be just for that Until @benbcai suggested better way to fix this issue. 😄

Reply:
@supreethmr Is data-date-time-input still needed or can it be removed?

Reply:
I don't we need data attribute in date-time-picker I will remove this.